ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -550,6 +550,8 @@ objects:
 
         wait "${openshift_install}"
 
+        mock-nss.sh openshift-install --dir=/tmp/artifacts/installer --log-level=debug wait-for install-complete 2>&1 &
```

Comment:
Isn't this already covered by the `wait "${openshift_install}"` above?

Reply:
Like all commits, a description for _why_ this is needed should be included in the commit message.

Reply:
The reason for adding this is - install is taking a little longer than the default timeout(30mins), so added this wait condition to add that extra wait time for all the cluster operators to come online.

Reply:
What we are finding is that the 30 minute timeout in the installer is not sufficient in our CI environment for ppc64le. We are running just over or into around 35 minutes. So this is an additional wait.

Reply:
The 30 timeout missing the install completion by a few minutes is also something we've run into ~20% of recent nightly builds on s390x. We should definitely include an explanation in the commit description though.

Reply:
Is there something exceptionally slow about our test environment? If not, I'd expect customers to run into this as well.

Reply:
I think we are not the only one facing this issue when I did this search - https://search.svc.ci.openshift.org/?search=failed+to+initialize+the+cluster%3A+Working+towards+.*waiting+on+authentication&maxAge=24h&context=1&type=bug%2Bjunit&name=&maxMatches=5&maxBytes=20971520, I see at least 10+ such cases in past 24hrs.

and interesting discussion about the time taking for libvirt install here by installer team - https://github.com/openshift/installer/pull/3333#issuecomment-602787363:

_a dev workflow that takes > 30 minutes is not really useful, i think our time is better spent on seeing what's slow and fixing that. libvirt doesn't fall into the bucket of baremetal which supposedly takes > 30 minutes due to machine-post-time being very high._


With all this, IMO having an additional wait condition at least helps us getting more coverage than failing. What do you say?

Reply:
If a timeout increase is needed, that change belongs in the installer itself; not CI. Our CI doesn't do us much good if it isn't reflective of what our customers are doing. Can we quickly figure out why the increased timeout is necessary and then figure out where the fix belongs?

Reply:
From what I can tell, this wouldn't be fixed quickly. The discussion @mkumatag linked to led me to another PR: https://github.com/openshift/release/pull/7568 -- which you obviously saw b/c you approved it (@crawford) :D -- but it's new to me, and I do see that it was an attempt to fix setup and teardown slowness:
https://github.com/openshift/release/pull/7568/files#diff-1c02c7c476c95d66377474c542fdfb33R274-R276 -- which gets us to the same thing, IIUC: https://github.com/ironcladlou/openshift4-libvirt-gcp/blob/rhel8/tools/create-cluster#L77

But it looks like that's only if libvirt is running in gcp? And this is on a system in our lab (meaning: not in GCP). And we don't have any specialized scripts. :/

We're okay with taking that wait out, but it would be nice if we could leave it in, and just open an issue with the installer in the meantime. I'll definitely update the commit message with more detail if you're okay with leaving this.  LMK!

Reply:
Also, @manojnkumar said that "in theory" no customers should see this version of the installer. It's very much unsupported (dev only), just to make sure that's clear. That makes me okay with this approach.

Reply:
Oh right, the IPI flow is only going to be used in CI. Duh. Sorry everyone, my mind has been scrambled this week. :disappointed: 

This is fine as is, but let's not forget to investigate this further.

Reply:
Thanks @crawford! I think we can all say the same thing about our minds these past couple of weeks. :)